contributing/documentation/overview.rst

```diff
@@ -35,48 +37,82 @@ Let's imagine that you want to improve the installation chapter of the Symfony
 book. In order to make your changes, follow these steps:
 
 **Step 1.** Go to the official Symfony documentation repository located at
-`github.com/symfony/symfony-docs`_ and `fork the repository`_ to your personal
-account. This is only needed the first time you contribute to Symfony.
+`github.com/symfony/symfony-docs`_ and click on the **Fork** button to `fork the
+repository`_ to your personal account. This is only needed the first time you
+contribute to Symfony.
 
-**Step 2.** **Clone** the forked repository to your local machine (this
-example uses the ``projects/symfony-docs/`` directory to store the documentation;
-change this value accordingly):
+**Step 2.** **Clone** the forked repository to your local machine (this example
+uses the ``projects/symfony-docs/`` directory to store the documentation; change
+this value accordingly):
```

Comment:
I am not convinced that we should still describe the steps to clone the repository locally as the best solution. Imo it's much easier to just browse to the file you want to edit on GitHub, click the edit button and make the changes just inside the text editor provided by GitHub. Most of the times this should be sufficient and is a much more straightforward way to contribute.